test/helpers/lines_helper_test.rb

```diff
@@ -5,15 +5,16 @@ class LinesHelperTest < ActionView::TestCase
 
   describe 'convenience methods' do
     it 'should spit out an array of available lines' do
-      assert_equal lines, %w(DC MD VA)
+      assert_equal lines, %w(DC MD VA).map(&:to_sym)
     end
 
     it 'should return current line' do
       assert_nil current_line
```

Comment:
This test still passes, but it's worth noting it used to call a different function (what is now `current_line_display`) and thus is testing a different section of code. Maybe add another line `assert_nil current_line_display` to be thorough?